app/models/contact_ticket.rb

```diff
@@ -68,6 +70,10 @@ def validate_mail_name_connection
     end
   end
 
+  def email_qq
+    /[0-9]+@qq.com$/.match?(email)
```

Comment:
This still permits `abc1@qq.com`, if that's not what you want.

Reply:
yep - not sure to be honest - I've not seen the data. I'm tempted to go with this as a blanket for now and adjust if we need to.